examples/all_softwares.ini

```
@@ -1,16 +1,17 @@
-[FITTING]
-software: bumps
-          dfo
-          gradient_free
-          gsl
-          levmar
-          mantid
-          matlab
-          matlab_curve
-          matlab_opt
-          matlab_stats
-          minuit
-          ralfit
-          scipy
-          scipy_ls
-          scipy_go
+[FITTING]
+software: bumps
+          dfo
+          gradient_free
+          gsl
+	  horace

```

Mind if we change this tab to the equivalent in spaces?
```suggestion
          horace

```